tests/snap_test.go

```diff
@@ -0,0 +1,199 @@
+package tests
+
+import (
+	"log"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/canonical/matter-snap-testing/utils"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	otbrSnap     = "openthread-border-router"
+	otbrSetupApp = "openthread-border-router.otbr-setup"
+	otbrAgentApp = "openthread-border-router.otbr-agent"
+	otbrWebApp   = "openthread-border-router.otbr-web"
+
+	infraInterface = "INFRA_IF"
+)
+
+var (
+	start          = time.Now()
```

Comment:
The value is reset in `setup()` and used in two different places. Does it need to be global?

Reply:
`TestThreadNetworkFormation()` and `setup()` both use `start` variables, so it has been declared as a global variable.

Reply:
The declaration in `setup()` is obsolete.

Reply:
> `TestThreadNetworkFormation()` and `setup()` both use `start` variables, so it has been declared as a global variable.

It should not be done that way because `TestThreadNetworkFormation` is a standalone test which starts the network and waits for it (by looking inside the logs). The results will be unexpected if this particular test is repeated or if another test ends up producing the same log. The current time should be collected right before the network startup.